.github/workflows/run_unit_tests.yml

```diff
@@ -71,26 +71,6 @@ jobs:
         run: poetry run pytest ./tests/unit -v --cov=ni_measurement_plugin_sdk_service --junitxml=test_results/nims-${{ matrix.os }}-py${{ matrix.python-version}}-all-extras.xml
         working-directory: ./packages/service
 
-      # ni-measurement-plugin-sdk-generator
```

Comment:
Do not remove this. Change it to run only the unit tests. The test you are adding in https://github.com/ni/measurement-plugin-python/pull/892 is a unit test.

Reply:
I have updated to unit tests.

Reply:
Ok, you have a path forward for removing mocking from acceptance tests, so you don't need to add unit tests.